src/components/TodoInfo/TodoInfo.tsx

```diff
@@ -1 +1,30 @@
-export const TodoInfo = () => {};
+import React from 'react';
+
+import { UserInfo } from '../UserInfo';
+import { Todo } from '../../App';
```

Comment:
[CODE STYLE] - According to the checklist, we should prepare data in one place (App component) and pass it to child components. Avoid using the `import` keyword in non-root components to import data. If `Todo` type is just a type and not importing any data, then this is fine; otherwise, consider refactoring to comply with the checklist.